doc-content/enterprise-only/admin-and-config/managing-business-central-artifacts-con.adoc

```diff
@@ -1,11 +1,11 @@
 [id='managing-business-central-artifacts-con']
-= Managing artifacts
+= Artifact management
 
 You can manage artifacts from the *Artifacts* page in {CENTRAL}. The artifact repository is a local Maven repository and there is only one Maven repository for each installation. {CENTRAL} recommends using Maven repository solutions like _Sonatype Nexus(TM)_, _Apache Archiva(TM)_, or _JFrog Artifactory(TM)_.
 
-The *Artifacts* page lists all the artifacts in the Maven repository. You can also upload a new artifact to the Maven repository using the *Upload* button.
+The *Artifacts* page lists all the artifacts in the Maven repository. You can also upload an artifact to the Maven repository using the *Upload* button.
```

Comment:
"Click *Upload* to upload an artifact to the Maven repository."
I like more imperative sentences, but that is a style choice I leave up to you. It does use far fewer words and is easier to translate.

Reply:
As this is a concept section, how about "You can click *Upload* to upload an artifact to the Maven repository."?

Reply:
As this is a concept section, I've opted for "You can upload artifacts to the Maven repository."